conda_forge_tick/contexts.py

```diff
@@ -55,6 +55,13 @@ def git_repo_owner(self) -> str:
     def git_repo_name(self) -> str:
         return f"{self.feedstock_name}-feedstock"
 
+    @property
+    def git_href(self) -> str:

```

Comment:
Maybe `get_http_ref` here or something? `get_href` confused me when I first read it.

Reply:
agreed. 0ffe783fd9c3038043d9e4986cb83f01edcdfceb